mmdet3d/datasets/convert_utils.py

```diff
@@ -253,16 +253,16 @@ def get_kitti_style_2d_boxes(info: dict,
         # transform the center from [0.5, 1.0, 0.5] to [0.5, 0.5, 0.5]
         dst = np.array([0.5, 0.5, 0.5])
         src = np.array([0.5, 1.0, 0.5])
-        loc = loc + dim * (dst - src)
-        loc_3d = np.copy(loc)
-        gt_bbox_3d = np.concatenate([loc, dim, rot], axis=1).astype(np.float32)
+        loc_center = loc + dim * (dst - src)
```

Comment:
suggest adding some comments in the modification, e.g., here we use gravity center here

Reply:
comments added